pylint/extensions/code_style.py

```diff
@@ -41,12 +48,39 @@ class CodeStyleChecker(BaseChecker):
             "Emitted where an in-place defined ``list`` can be replaced by a ``tuple``. "
             "Due to optimizations by CPython, there is no performance benefit from it.",
         ),
+        "R6103": (
+            "Use '%s' instead",
+            "consider-using-assignment-expr",
+            "Emitted when an if assignment is directly followed by an if statement and "
+            "both can be combined by using an assignment expression ``:=``. "
+            "Requires Python 3.8",
+        ),
     }
+    options = (
+        (
+            "max-line-length-suggestions",
+            {
+                "default": 88,
```

Comment:
I think the default should be ``max-line-length`` from the format checker. This is the way to make pylint compatible with black formatting (the line length can be changed in black).